src/integration-test/java/uk/gov/di/ipv/cri/passport/integrationtest/DataStoreIT.java

```diff
@@ -0,0 +1,115 @@
+package uk.gov.di.ipv.cri.passport.integrationtest;
+
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient;
+import com.amazonaws.services.dynamodbv2.document.DynamoDB;
+import com.amazonaws.services.dynamodbv2.document.Item;
+import com.amazonaws.services.dynamodbv2.document.Table;
+import org.junit.jupiter.api.Test;
+import uk.gov.di.ipv.cri.passport.domain.DcsResponse;
+import uk.gov.di.ipv.cri.passport.domain.PassportFormRequest;
+import uk.gov.di.ipv.cri.passport.persistence.DataStore;
+import uk.gov.di.ipv.cri.passport.persistence.item.AuthorizationCodeItem;
+import uk.gov.di.ipv.cri.passport.persistence.item.PassportCheckDao;
+
+import java.net.URI;
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DataStoreIT {
+
+    private static final String dynamodbUrl = "http://localhost:4567";
+    private static final URI endpoint = URI.create(dynamodbUrl);
+    private static final String DCS_RESPONSE_TABLE_NAME = "local-dev-dcs-response";
+    private static final String DCS_AUTHORIZATION_TABLE_NAME = "local-dev-auth-codes";
+    private static final DataStore<PassportCheckDao> dcsResponseDataStore =
+            new DataStore<>(
+                    DCS_RESPONSE_TABLE_NAME, PassportCheckDao.class, DataStore.getClient(endpoint));
+
+    private static final AmazonDynamoDB independentClient =
+            AmazonDynamoDBClient.builder()
+                    .withEndpointConfiguration(
+                            new AwsClientBuilder.EndpointConfiguration(dynamodbUrl, "eu-west-2"))
+                    .build();
+    private static final DynamoDB localDynamoDb = new DynamoDB(independentClient);
+    private static final Table dcsResponseTableHarness =
+            localDynamoDb.getTable(DCS_RESPONSE_TABLE_NAME);
+    private static final Table authorizationCodeTableHarness =
+            localDynamoDb.getTable(DCS_AUTHORIZATION_TABLE_NAME);
+
+    @Test
+    public void shouldPutPassportCheckIntoTable() {
+        DcsResponse validDcsResponse =
+                new DcsResponse(
+                        UUID.randomUUID(),
+                        UUID.randomUUID(),
+                        false,
+                        true,
+                        new String[] {"No errors here"});
```

Comment:
I think this might be bette as being `null`. I believe that's what DCS will respond with if there are no errors.

Reply:
Done